assets/src/design-system/components/index.js

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
-export { Text, Display, Headline } from './typography';
-export * from './button';
+export { Button } from './button';
 export { Modal } from './modal';
+export * as Snackbar from './snackbar';
```

Comment:
Chose to export this way so it is abundantly clear how snackbar structure works and flexible for the different functionality at play in dashboard and editor.
```
<Snackbar.Container>
<Snackbar.Message /> 
</Snackbar.Container> 
```